dbt_project.yml

```diff
@@ -1,4 +1,4 @@
 name: 'fivetran_utils'
-version: '0.4.1'
+version: '0.4.2'
```

Comment:
0.4.1 isn't released yet, we can keep this as 0.4.1

Reply:
updated